xml/cap_depl_install_minimal.xml

```diff
@@ -462,6 +462,9 @@ kube:
     password: ""
   organization: "cap"
   auth: "rbac"
+  psp:
+    nonprivileged: <replaceable>"suse.cap.psp"</replaceable>
+    privileged: <replaceable>"suse.cap.psp"</replaceable>
```

Comment:
I don't believe the minimal install section has mentioned [PSPs](https://upload.wikimedia.org/wikipedia/commons/4/46/Psp-1000.jpg) at all, so I'm not sure where `suse.cap.psp` would come from.

Reply:
Hmm, you're right. It should though since the minimal install suggests using CaaSP. I'll add something to point back to the `SUSE CaaS Platform 3 PodSecurityPolicy` section